packages/buy-button/src/__tests__/BuyButton.test.js

```diff
@@ -1,16 +1,40 @@
 import React from 'react'
 import { shallow } from 'enzyme'
 
-import BuyButton from '../index'
+import { BuyButton } from '../index'
 
 describe('<BuyButton />', () => {
-  it('should pass', () => {
-    expect(1).toBe(1)
+  let wrapper
+
+  let val = 2
+  const valExpected = 3
+
+  const props = {
+    quantity: 1,
+    skuId: 1,
+    seller: '1',
+    salesChannel: '1',
+    orderFormId: '12321',
+    mutate: jest.fn(),
+    afterClick: function() {
```

Comment:
Use `jest.fn()` to check if the function is called

Reply:
👍